packages/next/build/index.ts

```diff
@@ -640,7 +658,9 @@ export default async function build(dir: string, conf = null): Promise<void> {
         })
 
         if (useStatic404) {
-          defaultMap['/_errors/404'] = { page: '/_error' }
+          defaultMap['/_errors/404'] = {
```

Comment:
With this new change can't we just make this `/404` and update @now/next?

```suggestion
          defaultMap['/404'] = {
```

Reply:
Until `pages404` is no longer experimental we still have the potential conflict for `pages/404.js`. After it's no longer experimental we can rename this from `/_errors/404` to `/404` if desired

Reply:
Okay!